packages/common/src/scopeSupportFacets/scopeSupportFacets.types.ts

```diff
@@ -118,6 +134,10 @@ const scopeSupportFacets = [
   "type.class",
   "type.typeArgument",
   "type.typeArgument.iteration",
+  "type.typeArgument.method",
+  "type.typeArgument.method.iteration",
+  "type.typeArgument.constructor",
+  "type.typeArgument.constructor.iteration",
```

Comment:
Not sure I understand what these are supposed to be

Reply:
Eg `new MyClass()`

Reply:
Copied the wrong thing. Updated now.